index.html

```diff
@@ -78,17 +75,6 @@
       </div>
     </header>
 
-    <!-- Announcement -->
-    <section class="bg-light" id="announcement">
-      <div class="container">
-        <div class="row">
-          <div class="col-lg-12 text-center">
-            <h2 class="section-heading">Deadline for applications is Feb 24th!</h2>
```

Comment:
Suggestion: update this banner to say applications are closed rather than remove it entirely.